x-pack/test/osquery_cypress/fleet_server.ts

```diff
@@ -12,7 +12,7 @@ import {
   startFleetServer,
 } from '@kbn/security-solution-plugin/scripts/endpoint/common/fleet_server/fleet_server_services';
 import { Manager } from './resource_manager';
-import { getLatestAvailableAgentVersion } from './utils';
+// import { getLatestAvailableAgentVersion } from './utils';
```

Comment:
nit: It's better to remove code instead of commenting it. Even when you plan to uncomment it in a next PR it may never happen.

Reply:
Thank you Maxim, in general I agree, but I really hope that this will get back in place ASAP, we don't want a hardcoded version here forever ;p 
But anyway: I hope you don't mind if we keep it this way one, I want to get this merged asap since it's FF day.